CASC/PortfolioMode.cpp

```diff
@@ -237,65 +237,37 @@ bool PortfolioMode::prepareScheduleAndPerform(const Shell::Property& prop)
     }
   };
 
-  // now various ways of creating schedule extension based on their "age and flavour"
-  if (env.options->schedule() == Options::Schedule::CASC) {
-
-    schedule.loadFromIterator(main.iterFifo());
-    schedule.loadFromIterator(fallback.iterFifo());
-    addScheduleExtra(main,schedule,"si=on:rtra=on");
-    addScheduleExtra(fallback,schedule,"si=on:rtra=on");
-
-  } else if (env.options->schedule() == Options::Schedule::CASC_SAT) {
-
-    schedule.loadFromIterator(main.iterFifo());
-    schedule.loadFromIterator(fallback.iterFifo());
-    // randomize and use the new fmb option
-    addScheduleExtra(main,schedule,"si=on:rtra=on");
-    addScheduleExtra(fallback,schedule,"si=on:rtra=on");
-
-  } else if (env.options->schedule() == Options::Schedule::SMTCOMP) {
-    // Normally we do main fallback main_extra fallback_extra
-    // However, in SMTCOMP mode the fallback is universal for all
-    // logics e.g. it's not very strong. Therefore, in SMTCOMP
-    // mode we do main main_extra fallback fallback_extra
-
+  if (env.options->schedule() == Options::Schedule::SMTCOMP) {
```

Comment:
Just for backwards-compatibility, right? No inherent reason we should do this for SMT-COMP.